src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
Agreed, this can be more simple implementation. Let me try again!
Thanks for your comments.